### PR TITLE
Add make:observe command to artisan

### PR DESCRIPTION
**Pull request by cracki**
The make:observe command will generate an observe class with these methods:

`creating, created, updating, updated, saving, saved, deleting, deleted, restoring, restored`

If 'Observers' directory does not exist, this command will create it.

### CODE REVIEW

File: src/Illuminate/Foundation/Console/ObserveMakeCommand.php [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+     * @var string
+     */
+    protected $name = 'make:observe';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Create a new observe class';
```
Comment (lucasmichot) [inline src/Illuminate/Foundation/Console/ObserveMakeCommand.php]: `Create a new model observer class`

File: src/Illuminate/Foundation/Console/ObserveMakeCommand.php [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+    protected $type = 'Observe';
+
+    /**
+     * Execute the console command.
+     *
+     * @return void
+     */
+    public function fire()
+    {
+        if (! $this->option('model')) {
```
Comment (cracki) [inline src/Illuminate/Foundation/Console/ObserveMakeCommand.php]: yes, fixed 👍

File: src/Illuminate/Foundation/Console/ObserveMakeCommand.php (hunk 1 above)
Comment (cracki) [inline src/Illuminate/Foundation/Console/ObserveMakeCommand.php]: fixed

**Comment by GrahamCampbell** [comment]
Please send to 5.4. :)

**Comment by GrahamCampbell** [comment]
Things have moved about.

File: src/Illuminate/Foundation/Console/ObserveMakeCommand.php (hunk 2 above)
Comment (lucasmichot) [inline src/Illuminate/Foundation/Console/ObserveMakeCommand.php]: So this check can be removed now? As the argument is required

Comment (cracki) [inline src/Illuminate/Foundation/Console/ObserveMakeCommand.php]: I think check need anyway because the user may not enter the "model" option

**Pull request closed without merging**